src/components/Login/Login.tsx

```diff
@@ -16,37 +16,23 @@ const Login = (): JSX.Element => {
     <LoginStyled>
       <Formik
         initialValues={initialValues}
-        validate={(values) => {
-          const errors: ErrorsHandler = {};
-          if (!values.username) {
-            errors.username = "Required username";
-          }
-
-          if (!values.password) {
-            errors.password = "Required password";
-          }
-
-          return errors;
-        }}
+        validationSchema={validateLoginSchema}
         onSubmit={(values, { resetForm }) => {
-          setTimeout(() => {
-            console.log(values);
-            resetForm();
-          }, 2000);
+          resetForm();
         }}
       >
-        {({ errors, isSubmitting }) => (
-          <Form autoComplete="off" className="form">
+        {({ errors, isValid }) => (
+          <Form autoComplete="off" className="form" noValidate>
             <div className="input-container">
               <label htmlFor="username" className="form-label">
```

Comment:
You can create a custom component that wraps the <label> and the <Field> tabs so the form code looks cleaner and its faster to write.

```
const FormField = ({type, name, id, label, placeholder... =})=>{
   return(
       <>
          <label htmlfor={id} ...> {label} </label>
          <Field {all props}/>
       </>
   )
}
```

Reply:
I just fixed it on this new pull